project/src/main/webapp/candidate.html

```diff
@@ -8,9 +8,20 @@
     <link rel="stylesheet" href="style.css">
     <script src="config.js"></script>
     <script src="script.js"></script>
+    <script src="common.js"></script>
     <title>Voter Central Candidate Page</title>
   </head>
   <body>
-    <h2>Voter Central Candidate Page</h3>
+    <div id="header-container"></div>
+    <div id="navbar-container"></div>
+    <h2>Voter Central Candidate Page</h2>
+    <div id="content">
```

Comment:
`<div>` doesn't have enough semantic meaning. Maybe change it to `<main>`.

Reply:
Just a reminder: the corresponding CSS styling rule would need to change as well